### PR TITLE
New package: promtail-2.6.1

### PR DESCRIPTION
**Pull request by shanduur**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64
  - aarch64-musl
  - aarch64

### CODE REVIEW

**Comment by Shanduur** [comment]
Template copies varlog config which assumes all logs can be found acording to the `/var/log/*log` pattern. Assuming *socklog-unix* and *svlogd* are used, `/var/log/socklog/*/current` pattern should be used.

File: srcpkgs/promtail/files/promtail/log/run [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec logger -p daemon.info -t promtail
```
Comment (Duncaen) [inline srcpkgs/promtail/files/promtail/log/run:2]: use `vlogger`.

File: srcpkgs/promtail/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+wrksrc="loki-${version}"
+build_style=go
+go_import_path="github.com/grafana/loki"
+go_package="${go_import_path}/clients/cmd/promtail"
+short_desc="Promtail is an agent which ships the contents of local logs to a Grafana Loki instance"
+maintainer="Mateusz Urbanek <murbanek@shanduur.com>"
+license="Apache-2.0"
+homepage="https://grafana.com/oss/loki/"
+distfiles="https://github.com/grafana/loki/archive/v${version}.tar.gz"
+checksum=4b41175e552dd198bb9cae213df3c0d9ca8cacd0b673f79d26419cea7cfb2df7
```
Comment (sgn) [inline srcpkgs/promtail/template:14]:
```suggestion
checksum=4b41175e552dd198bb9cae213df3c0d9ca8cacd0b673f79d26419cea7cfb2df7
conf_files=/etc/promtail/promtail-local-config.yaml
```

File: srcpkgs/promtail/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'promtail'
+pkgname=promtail
+version=2.6.1
+revision=1
+wrksrc="loki-${version}"
+build_style=go
+go_import_path="github.com/grafana/loki"
+go_package="${go_import_path}/clients/cmd/promtail"
+short_desc="Promtail is an agent which ships the contents of local logs to a Grafana Loki instance"
```
Comment (sgn) [inline srcpkgs/promtail/template:9]:
```suggestion
short_desc="Agent to ship the contents of local logs to a Grafana Loki instance"
```

File: srcpkgs/promtail/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/promtail/template:14]: Anyway, what's wrong with just make the file named: `/etc/promtail/config.yaml`?

Comment (Shanduur) [reply srcpkgs/promtail/template:14]: @sgn  will adding just `conf_files=/etc/promtail/config.yaml` create empty file? Or do I need:

```diff
-	vcopy clients/cmd/promtail/promtail-local-config.yaml etc/promtail/promtail-local-config.yaml
+	vcopy clients/cmd/promtail/promtail-local-config.yaml etc/promtail/config.yaml
```

File: srcpkgs/promtail/files/promtail/run [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+[ -f ./conf ] && . ./conf
+
+export ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.19
+
+exec promtail \
+    --config.file=/etc/promtail/promtail-local-config.yaml \
```
Comment (Duncaen) [inline srcpkgs/promtail/files/promtail/run:8]: This is the wrong path, if we keep shipping the "default" configuration file.

**Comment by Shanduur** [comment]
> I don't really like how the service invents a new "default" configuration file and its location, this always leads to issues when upstream sets their own default.

So what do you propose? Is there a way to include that config file into the tree, so the "default" won't change?

**Pull request closed without merging**